mobile_codes/json/mnc_operators.json

```diff
@@ -1,1344 +1,1933 @@
 [
-    ["001", "01", "TEST", "Test Network"],
-    ["202", "01", "Cosmote", "Cosmote"],
-    ["202", "02", "Cosmote", "Cosmote"],
-    ["202", "03", "OTE", "OTE"],
-    ["202", "04", "EDISY", "EDISY"],
-    ["202", "05", "Vodafone - Panafon", "Vodafone - Panafon"],
-    ["202", "06", "COSMOLINE", "COSMOLINE"],
-    ["202", "07", "AMD TELECOM", "AMD TELECOM"],
-    ["202", "09", "WIND", "WIND"],
-    ["202", "10", "WIND", "WIND"],
-    ["204", "01", "RadioAccess Network Services B.V.", "RadioAccess Network Services B.V."],
-    ["204", "02", "Tele2 Nederland B.V.", "Tele2 Nederland B.V."],
-    ["204", "03", "Voiceworks B.V.", "Voiceworks B.V."],
-    ["204", "04", "Vodafone Libertel B.V.", "Vodafone Libertel B.V."],
-    ["204", "05", "Elephant Talk Communications Premium \nRate Services", "Elephant Talk Communications Premium \nRate Services"],
-    ["204", "06", "Mundio Mobile (Netherlands) Ltd", "Mundio Mobile (Netherlands) Ltd"],
-    ["204", "07", "Teleena holding B.V.", "Teleena holding B.V."],
-    ["204", "08", "KPN Mobile The Netherlands B.V.", "KPN Mobile The Netherlands B.V."],
-    ["204", "09", "Lycamobile Netherlands Limited", "Lycamobile Netherlands Limited"],
-    ["204", "10", "KPN B.V.", "KPN B.V."],
+    ["001", "001", "TEST", "Test network"],
+    ["001", "01", "TEST", "Test network"],
+    ["202", "01", "Cosmote", "COSMOTE - Mobile Telecommunications S.A."],
+    ["202", "02", "Cosmote", "COSMOTE - Mobile Telecommunications S.A."],
+    ["202", "03", "", "OTE"],
+    ["202", "04", "", "OSE"],
+    ["202", "05", "Vodafone", "Vodafone Greece"],
+    ["202", "06", "", "Cosmoline"],
+    ["202", "07", "", "AMD Telecom"],
+    ["202", "09", "Wind", "Wind Hellas Telecommunications S.A."],
+    ["202", "10", "Wind", "Wind Hellas Telecommunications S.A."],
+    ["202", "11", "", "interConnect"],
+    ["202", "12", "", "Yuboto"],
+    ["202", "13", "", "Compatel Limited"],
+    ["202", "14", "Cyta Hellas", "CYTA"],
+    ["202", "15", "", "BWS"],
+    ["202", "16", "", "Inter Telecom"],
+    ["204", "01", "", "RadioAccess Network Services"],
+    ["204", "02", "Tele2", "Tele2 Nederland B.V."],
+    ["204", "03", "Voiceworks", "Voiceworks B.V."],
+    ["204", "04", "Vodafone", "Vodafone Libertel B.V."],
+    ["204", "05", "", "Elephant Talk Communications Premium Rate Services"],
+    ["204", "06", "Vectone Mobile", "Mundio Mobile (Netherlands) Ltd"],
+    ["204", "07", "", "Teleena (MVNE)"],
+    ["204", "08", "KPN", "KPN Mobile The Netherlands B.V."],
+    ["204", "09", "Lycamobile", "Lycamobile Netherlands Limited"],
+    ["204", "10", "KPN", "KPN B.V."],
     ["204", "11", "", "VoipIT B.V."],
-    ["204", "12", "KPN B.V.", "KPN B.V."],
-    ["204", "13", "Unica Installatietechniek B.V.", "Unica Installatietechniek B.V."],
+    ["204", "12", "Telfort", "KPN Mobile The Netherlands B.V."],
+    ["204", "13", "", "Unica Installatietechniek B.V."],
```

Comment:
This means there are more blank brands, when the [Wikipedia source](https://en.wikipedia.org/wiki/Mobile_Network_Codes_in_ITU_region_2xx_(Europe)#Netherlands_(Kingdom_of_the_Netherlands)_–_NL) has an empty Brand column

Reply:
Yes. But if somehow (?) this is causes any issue the parser script can be adapted to use the operator value in case the brand is an empty string